object_detection/BoundingBoxes[12] detected_object
sensor_msgs/Image img
